stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/attachments/content/MediaAttachmentContent.kt

```diff
@@ -337,7 +338,10 @@ internal fun MediaAttachmentContentItem(
 
     val data =
         if (isImage || (isVideo && ChatTheme.videoThumbnailsEnabled)) {
-            attachment.imagePreviewUrl
+            when (message.syncStatus) {
+                SyncStatus.COMPLETED -> attachment.imagePreviewUrl
```

Comment:
Using `Attachment.uploadState` here seemsto be slightly more error prone because it sometimes gets cleared if we kill the app, plus this is what `MessageBubble` uses so this way they are in sync when it comes to the general state.

Also using inner `when` here instead of an if-else because I think it has a lower cognitive load than nesting an if-else inside an if. Just trying to lower repetition.